### PR TITLE
[new plugin] markdowny.nvim

### PR DESCRIPTION
**Pull request by antonk52**

### CODE REVIEW

**Comment by phaazon** [comment]
Please target the right branch. I’m changing the base branch.

File: contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <i class="fa-solid fa-book"></i>
+      </span>
+    </span>
+    <span>markdowny.nvim</span>
+  </a>
+</h3>
+
+![markdowny.nvim](https://user-images.githubusercontent.com/5817809/211911652-fe0c1d26-1dd0-4832-b948-e685067bb78b.gif)
+
+A new plugin by @antonk52 to add markdown like shortcuts similar to Reddit/Notion/Slack/Discord etc
```
Comment (phaazon) [inline contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md:14]: Add a link to your GitHub page on `@antonk52`.

Comment (phaazon) [reply contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md:14]: Also, please use _Markdown_ instead of _markdown_.

Comment (phaazon) [reply contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md:14]: And dot at the end of the sentence.

**Review by phaazon — CHANGES_REQUESTED** [review]

File: contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md (hunk 1 above)
Comment (antonk52) [reply contents/2023/Jan/16/3-new-plugins/1-markdowny.nvim.md]: Thanks for the review! Fixed.

**Pull request merged**